### ISSUE TITLE
OPTICKS-559 - Exporting movie of SDV in product causes data to zoom out

### ISSUE THREAD

**Issue by tclarke** [opticks-org/opticks#116]
1. Load a non-square data cube
 2. Create an animation for it
 3. Derive a product view from it
 4. On Product 1 / Layout / View 1 / DataSetName select Export
 5. Choose MPEG or AVI exporter and export the movie
 6. As it exports, note that the data is slowly zooming out
 7. Play the movie back in media player
 8. As the movie plays, note that the zoom-out shows up in the movie as well (as expected based on seeing it in step 6)

Original Issue: [OPTICKS-559](https://issues.opticks.org/jira/browse/OPTICKS-559)
Status: Deferred
Feature Interpretation:

Suggested Solution:
This will require some rethinking about how we are currently exporting a movie file. The current way Opticks is doing this is by using the ViewImp::getCurrentImage method to get a frame for the movie. However, this method uses zooming to get rid of width and height issues. It's this zooming that causing this issue to occur when exporting a movie in product view. There are at least three approaches we can implement to get around this problem, but none of them will most likely solve this issue and may even be more trouble then what its worth, definitely when OPTICKS-445 (Add capability to position layers in a view according to geocoordinates) may require changing how views work in Opticks. I recommend that we wait for the implementation of OPTICKS-445 before we can continue to look into this issue further.

Actual Solution:

Severity: 3-Important
Version Control: 
Environment: All
Linked:
